x-pack/legacy/plugins/index_management/public/sections/home/templates_list/templates_table/templates_table.tsx

```diff
@@ -34,6 +43,17 @@ export const TemplatesTable: React.FunctionComponent<Props> = ({ templates, relo
       }),
       truncateText: true,
       sortable: true,
+      render: (name: Template['name']) => {
+        return (
+          <EuiLink
+            href={`#${BASE_PATH}templates/${name}`}
+            data-test-subj="templateDetailsLink"
+            onClick={trackUiMetric.bind(null, UIM_TEMPLATE_SHOW_DETAILS_CLICK)}
```

Comment:
this is changed now, its gonna be `trackUiMetric('click', UIM_TEMPLATE_SHOW_DETAILS_CLICK)`

Reply:
fixed, thanks for catching this!